docs/documentation/maya/simshape.md

```diff
@@ -195,14 +229,16 @@ The AMP file is generated from the Learn Muscle Patches tool:
 
 Additional custom settings for the learning algorithm:
 
-| Settings                   | Type  | Value     |  Range/Options | Description                  |
-| :------------              | :---  | :----     |  :------------ | :--------------------------- |
-| Limit Iterations           | Bool  | False     |                | If enabled, the "Number of iterations" will be taken into consideration. |
-| Number of Iterations       | Int   | 20        | \[1, 1e^6^\]   | Maximum number of iterations allowed in the training process. The higher this value is, the more accurate the muscle patches estimation will be and the longer the execution will take. This parameter is ignored if "Limit iterations" attribute is disabled. In that case, the training process will run until it achieves the most accurate solution. |
-| Number of Muscle Patches   | Int   | 79        | \[1, 1e^6^\]   | Maximum number of muscle patches expected in the results. |
-| Draw Muscle Patches        | Bool  | True      |                | If enabled, the vertices of the neutral mesh will be colored according to the muscle patches resulting from the training. |
+ - **Limit Iterations** (Boolean, False): If enabled, the "Number of iterations" will be taken into consideration.
```

Comment:
I see that sometimes to make reference to an attribute it is wrapped in quotation marks, other times italic style is used and other times bold style, I think we should follow the same criteria to reference attributes.

Reply:
You are right. I believe out of these three options the most adequate is the italic style. We shoul do in the near future a check through all pages to be consistent with this.

Reply:
Removed bold or italic style from references to attributes.